### DISCUSSION TITLE
Accepting pull requests before release

### DISCUSSION THREAD

**Discussion by oyvindseland** [NorESMhub/CAM#65] [Team Posts]
Should we go briefly through the pull requests mid next week so that we can agree on them.

**Comment by oyvindseland** [comment]
or should we just accept them ourselves if the change is accepted by the reviewer(s) ?

**Comment by DirkOlivie** [comment]
There are two remaining pull requests in CAM (one on **redundant code**, and one on **a preliminary contribute.md**).  Can they be in the upcoming release?

**Comment by MichaelSchulzMETNO** [comment]
I merged the contribute.md, since that was just for all of you to see and possible edit in case it was not ok. For the redundant code, I am confused and added a remark on the pull request.

**Comment by DirkOlivie** [comment]
I did a test on vilje with the redundant code changes, and it did not affect the results (tested NF1850norbc compset).

**Comment by oyvindseland** [comment]
Outdated discussion
